submissions/emmanuelEsuong/htmlTask/index.html

```diff
@@ -0,0 +1,161 @@
+<!DOCTYPE html>
+
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Welcome To GreenEatz Online Store</title>
+</head>
+
+<body>
+<!--GreenEat Head logo-->
+       <pre>
+                                                                            <img src="GreenEats_logo.png" alt="image"></img>
+       </pre>
+
+            <table>
+                <tr>
+                        <td><a href="#"><b>HOME</a></td>
+
+                        <td><a href="#"><b>ABOUT US</a></td>
+
+                        <td><a href="#"><b>BUSINESS UNITS</a></td>
+
+                        <td><a href="#"><b>FOODCO FOUNDATION</a></td>
+
+                        <td><a href="#"><b>CAREERS</a></td>
+
+                        <td><a href="#"><b>LOCATIONS</a></td>
+
+                        <td><a href="#"><b>WHAT'S NEW</a></td>
+
+                        <td><a href="#"><b>CONTACT US</a></td>
+
+                        <td><a href="#"><b>PROMOS</a></td>
+                 <tr>
+           </table>
+       
+    <pre>
+
+    </pre>
+        
+        <!--Products/Menu list-->   
+        <pre>
+
+        </pre>
+    
+        <pre>
+        <p></pre>
+            Shop at the Best Prices and Get Your Order Delivered at Your Door-Step in a Jiffi
+        </pre></p>
+        
+
+        <form action="/form/submit" method="GET"><input type="text" name="text" class="search" placeholder="Search here!"><input type="submit" name="submit" class="submit" value="Search"</form>
```

Comment:
You need to remove the action and method element

Reply:
Element removed.